chrome/content/zotero/zoteroPane.js

```diff
@@ -4440,6 +4440,76 @@ var ZoteroPane = new function()
 			this.loadURI(supportURL, { metaKey: true, shiftKey: true });
 		}
 	}
+
+
+	/**
+	 * Prompt the user to relink one or all of the attachment files found in
+	 * the LABD.
+	 *
+	 * @param {Zotero.Item} item
+	 * @param {String} path Path to the file matching `item`
+	 * @param {Number} numOthers If zero, "Relink All" option is not offered
+	 * @return {'one'|'all'|false}
+	 */
+	this.showAttachmentFoundAutomaticallyDialog = function (item, path, numOthers) {
+		let title = Zotero.getString('pane.item.attachments.autoRelink.title');
+		let text = Zotero.getString('pane.item.attachments.autoRelink.text1')
+			+ '\n\n'
+			+ path
+			+ '\n\n'
+			+ Zotero.getString(
+				'pane.item.attachments.autoRelink.text2.'
+					+ (numOthers ? 'others' : 'noOthers'),
+				[Zotero.appName, numOthers],
+				numOthers
+			);
+
+		let ps = Services.prompt;
+
+		if (numOthers) {
+			let buttonFlags = ps.BUTTON_POS_0 * ps.BUTTON_TITLE_IS_STRING
+				+ ps.BUTTON_POS_1 * ps.BUTTON_TITLE_IS_STRING
+				+ ps.BUTTON_POS_2 * ps.BUTTON_TITLE_IS_STRING;
+			let index = ps.confirmEx(null,
+				title,
+				text,
+				buttonFlags,
+				Zotero.getString('pane.item.attachments.autoRelink.relinkAll'),
+				Zotero.getString('pane.item.attachments.autoRelink.relinkOne'),
+				Zotero.getString('pane.item.attachments.autoRelink.locateManually'),
```

Comment:
The way the Mozilla confirmation dialog works, index 1 is what happens if you press Esc in the dialog, so that has to either be an explicit Cancel or the least-destructive option. A Cancel is usually nice to offer anyway. Can't have more than three buttons, so probably best to make `locateManually` index 1, and then you can always cancel out of the filepicker?